pkg/app/test_command.go

```diff
@@ -121,6 +126,23 @@ func convergeResults(result runtime.Result, new runtime.Result) runtime.Result {
 	return result
 }
 
+func testStdin(filters runtime.Filters) (runtime.Result, error) {
+	f, err := os.Stdin.Stat()
+	if err != nil {
+		return runtime.Result{}, err
+	}
+
+	if f.Mode()&os.ModeCharDevice != 0 {
```

Comment:
Very good catch! 
I am not sure if it was by intention to add the binary operation directly without spaces between it.
`if (f.Mode() & os.ModeCharDevice) != 0` is imho more clear about whats going on.

 At first I thought you referenced to the address `os.ModeCharDevice` by adding the `&` directly behind the variable. Looks a little bit confusing in the first moment.
You can install our `git-hooks` with `make init` which should check the coding style before you can commit. If you want to omit the hook just call `git commit -m "test" --no-verify`

Reply:
I see what you mean.